src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -462,7 +462,9 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
       } yield ()
     }
 
-    /** Groups data needed for generating Governance */
+    /** Groups data needed for generating GovernanceRow
+     *  Warning! It works on assumption that we're not processing Genesis block
+     */
```

Comment:
can be marked private?

Reply:
nope, it can't

Reply:
just to be clear, compiler does not allow me to add private to method within method